core/domain/draft_upgrade_services.py

```diff
@@ -105,7 +105,9 @@ class DraftUpgradeUtil(python_utils.OBJECT):
 
     @classmethod
     def _convert_states_v43_dict_to_v44_dict(cls, draft_change_list):
-        """Converts draft change list from state version 42 to 43.
+        """Converts draft change list from state version 42 to 43. State
+        version 44 adds a linked skill id variable to the
```

Comment:
variable --> property

You haven't explained *why* there should be no changes to drafts. You might want to explain that this is a new property and therefore doesn't affect any pre-existing drafts.

Reply:
Done.